### PR TITLE
Update VoiceOver navigation commands for Command and Toggle Button

### LINKED ISSUE #1030: Feedback: "Navigate backwards to a pressed toggle button" (Toggle Button, Test 12, V23.12.14)
## Description of Behavior

For the Shift+J test,  the output was "Run Test Setup, dimmed, button".  One of the assertions was that the role was Button.  The role was announced as button, but focus was on the wrong element. I got a conflict when I passed one assertion, but failed the other two.  This was the same for all other Shift+J tests.

## Test Setup

- Test File: [test-12-navBackToPressedToggleButton-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/toggle-button/test-12-navBackToPressedToggleButton-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 14.2)
- Browser: Safari (Version 17.2.1)

### PR DESCRIPTION
**Pull request by IsaDC**
[Preview Tests](https://deploy-preview-1048--aria-at.netlify.app)
Resolves #1030
- Update VoiceOver navigation commands for Command Button
- Update navigation VoiceOver commands for Toggle Button

### CODE REVIEW

**Comment by IsaDC** [comment]
Hi @howard-e could you please help me out with all these errors? Do I need to update something on my computer?
Thank you kindly.

File: tests/command-button/data/voiceover_macos-commands.csv [hunk 1]
```diff
@@ -1,8 +1,10 @@
 testId,command,settings,assertionExceptions,presentationNumber
 navForwardsToButton,ctrl+opt+right,,,5
+navForwardsToButton,b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/command-button/data/voiceover_macos-commands.csv:3]:
```suggestion
navForwardsToButton,b,singleQuickKeyNavOn,,5.3
```

File: tests/command-button/data/voiceover_macos-commands.csv [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 navForwardsToButton,tab,,,5.1
 navForwardsToButton,j ,singleQuickKeyNavOn,,5.2
 navBackToButton,ctrl+opt+left,,,6
+navBackToButton,shift+b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/command-button/data/voiceover_macos-commands.csv:7]:
```suggestion
navBackToButton,shift+b,singleQuickKeyNavOn,,6.3
```

File: tests/toggle-button/data/voiceover_macos-commands.csv [hunk 3]
```diff
@@ -1,14 +1,18 @@
 testId,command,settings,assertionExceptions,presentationNumber
 navForwardsToNotPressedToggleButton,ctrl+opt+right,,,5
+navForwardsToNotPressedToggleButton,b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/toggle-button/data/voiceover_macos-commands.csv:3]:
```suggestion
navForwardsToNotPressedToggleButton,b,singleQuickKeyNavOn,,5.3
```

File: tests/toggle-button/data/voiceover_macos-commands.csv [hunk 4]
```diff
@@ -1,14 +1,18 @@
...
 navForwardsToNotPressedToggleButton,tab,,,5.1
 navForwardsToNotPressedToggleButton,j ,singleQuickKeyNavOn,,5.2
 navBackToNotPressedToggleButton,ctrl+opt+left,,,6
+navBackToNotPressedToggleButton,shift+b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/toggle-button/data/voiceover_macos-commands.csv:7]:
```suggestion
navBackToNotPressedToggleButton,shift+b,singleQuickKeyNavOn,,6.3
```

File: tests/toggle-button/data/voiceover_macos-commands.csv [hunk 5]
```diff
@@ -1,14 +1,18 @@
...
 navBackToNotPressedToggleButton,shift+tab,,,6.1
 navBackToNotPressedToggleButton,shift+j ,singleQuickKeyNavOn,,6.2
 navForwardsToPressedToggleButton,ctrl+opt+right,,,11
+navForwardsToPressedToggleButton,b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/toggle-button/data/voiceover_macos-commands.csv:11]:
```suggestion
navForwardsToPressedToggleButton,b,singleQuickKeyNavOn,,11.3
```

File: tests/toggle-button/data/voiceover_macos-commands.csv [hunk 6]
```diff
@@ -1,14 +1,18 @@
...
 navForwardsToPressedToggleButton,tab,,,11.1
 navForwardsToPressedToggleButton,j ,singleQuickKeyNavOn,,11.2
 navBackToPressedToggleButton,ctrl+opt+left,,,12
+navBackToPressedToggleButton,shift+b,singleQuickKeyNavOn,,
```
Comment (howard-e) [inline tests/toggle-button/data/voiceover_macos-commands.csv:15]:
```suggestion
navBackToPressedToggleButton,shift+b,singleQuickKeyNavOn,,12.3
```

**Review by howard-e — COMMENTED** [review]
@IsaDC the `presentationNumber` value is missing for the newly added rows.

I've left inline suggestions for what those values could be just based on the other presentationValues around it, but please adjust if I'm wrong!

**Pull request merged**